cmd/rpcdaemon/commands/eth_api_xlayer.go

```diff
@@ -11,5 +16,45 @@ func (apii *APIImpl) GetGPCache() *GasPriceCache {
 func (apii *APIImpl) runL2GasPricerForXLayer() {
 	// set default gas price
 	apii.gasCache.SetLatest(common.Hash{}, apii.L2GasPricer.GetConfig().Default)
+	apii.gasCache.SetLatestRawGP(apii.L2GasPricer.GetConfig().Default)
 	go apii.runL2GasPriceSuggester()
 }
+
+const cacheSize = 30 // Circular buffer size
```

Comment:
Write some comments on how we determine cacheSize
```suggestion
// 1200sec (TTL) / 10sec (UpdatePeriod)
const cacheSize = 30
```